vip_hci/negfc/mcmc_sampling.py

```diff
@@ -325,7 +327,8 @@ def mcmc_negfc_sampling(cube, angs, psfn, ncomp, plsc, initial_state, fwhm=4,
                         ac_c=50, ac_count_thr=3, nproc=1, output_dir='results/', 
                         output_file=None, display=False, verbosity=0, 
                         save=False):
-    r""" Runs an affine invariant mcmc sampling algorithm in order to determine
```

Comment:
This is not a typo. It means it is a [raw string](https://www.codevscolor.com/python-raw-string)